invoke-ai/InvokeAI
# feat(ui): upscaling & ad-hoc post-processing misc

## Summary

Round of updates & fixes for upscaling tab and ad-hoc post-processing (previously called ad-hoc upscaling). Mostly fixing verbiage and splitting off the post-processing UI from the upscaling UI.

One other notable change:
- [feat(nodes): split spandrel node](https://github.com/invoke-ai/InvokeAI/commit/06560ca9e6b443942263ba4cea2d17f31aa9e91e)
    
    `spandrel_image_to_image` now just runs the model with no changes.
    
    `spandrel_image_to_image_autoscale` runs the model repeatedly until the desired scale is reached. previously, `spandrel_image_to_image` did this.

The ad-hoc post-processing feature uses `spandrel_image_to_image`. It processes the image using the model without any special handling. The upscaling tab uses `spandrel_image_to_image_autoscale` - its behaviour is unchanged.

## Related Issues / Discussions

n/a

## QA Instructions

The ad-hoc upscale popover UI is updated with its new identity as a post-processing popover.

<video src="https://github.com/user-attachments/assets/ecfc4fbf-a7b7-4932-9b45-c2400f6d49a8"></video>

Because we now refer to spandrel models as "image to image", "upscale" and "post-processing" models in different places, the starter models search is updated to understand these search terms.

<video src="https://github.com/user-attachments/assets/ffee1632-2571-4263-94f3-b7905dc4fedb"></video>

All generation functionality - both upscaling tab and post-processing popover - should be unchanged.

## Merge Plan

n/a

## Checklist
